components/create_post/index.js

```diff
@@ -93,6 +91,7 @@ function makeMapStateToProps() {
         });
         const channelMemberCountsByGroup = selectChannelMemberCountsByGroup(state, currentChannel.id);
         const currentTeamId = getCurrentTeamId(state);
+        const groupsWithAllowReference = useGroupMentions ? new Map() : getAssociatedGroupsForReferenceByMention(state, currentTeamId, currentChannel.id);
```

Comment:
Farhan, can you help me understand this? If we do not have `groupMentions` set, then we call `getAssociatedGroupsForReferenceByMention`?

Reply:
This was incorrect... good catch @catalintomai 😅